.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
+force_alphabetical_sort = True
```

Comment:
@Rotonen what do you mean by it does not work?
I just tried this snippet and it works quite well on my machine 😏